lib/formatters.js

```diff
@@ -109,12 +113,12 @@ var formatOutputUInt = function (value) {
 
 /// @returns input bytes formatted to real
 var formatOutputReal = function (value) {
-    return formatOutputInt(value).dividedBy(new BigNumber(2).pow(128)); 
+    return formatOutputInt(value).div(new BigNumber(2).toRed(BigNumber.red('k256')).redPow(new BigNumber(128)).fromRed()); 
```

Comment:
maybe define `const TWO_P256 = '0100000000000000000000000000000000'` somewhere. Then use  like `new BN(TWO_P256, 16) ` .  Sidenote, pows are the most expensive operation in the library.  I assume that why you can't use them if your not in `reduction` mode